Resumable Upload: Remove appendices (#2994)
httpwg/http-extensions: draft-ietf-httpbis-resumable-upload.md

A possible drawback is that the client might be unable to resume an upload. If an upload is interrupted before the client received a `104 (Upload Resumption Supported)` intermediate response with the upload URL, the client cannot resume that upload due to the missing upload URL. The intermediate response might not be received if the interruption happens too early in the message exchange, the server does not support resumable uploads at all, the server does not support sending the `104 (Upload Resumption Supported)` intermediate response, or an intermediary dropped the intermediate response. Without a 104 response, the client needs to either treat the upload as failed or retry the entire upload creation request if this is allowed by the application.

> A client might wait for a limited duration for the receival of the 104 intermediate response before starting to transmit the request content, similar to how a client might wait for the response of a "100-continue" expectation (see {{Section 10.1.1 of HTTP}}). This way, the client can learn about the resorce's support for resumable uplaods and/or the upload URL.

Reviewer: ```suggestion
A client might wait for a limit duration to receive a 104 (Upload Resumption Supported) interim response before starting to transmit the request content. This way, the client can learn about the resource's support for resumable uplaods and/or the upload URL. This is conceptually similar to how a client might wait for a 100 (Continue) interim response  (see {{Section 10.1.1 of HTTP}}) before comitting to work. 
```

> A client might wait for a limited duration to receive a 104 (Upload Resumption Supported) interim response before starting to transmit the request content. This way, the client can learn about the resource's support for resumable upload and/or the upload URL. This is conceptually similar to how a client might wait for a 100 (Continue) interim response  (see {{Section 10.1.1 of HTTP}}) before committing to work. 

Author: ```suggestion
A client might wait for a limited duration to receive a 104 (Upload Resumption Supported) interim response before starting to transmit the request content. This way, the client can learn about the resource's support for resumable uploads and/or the upload URL. This is conceptually similar to how a client might wait for a 100 (Continue) interim response (see {{Section 10.1.1 of HTTP}}) before committing to work. 
```

> A client might wait for a limited duration to receive a 104 (Upload Resumption Supported) interim response before starting to transmit the request content. This way, the client can learn about the resource's support for resumable uploads and/or the upload URL. This is conceptually similar to how a client might wait for a 100 (Continue) interim response (see {{Section 10.1.1 of HTTP}}) before committing to work. 

Author: ```suggestion
A client might wait for a limited duration to receive a 104 (Upload Resumption Supported) interim response before starting to transmit the request content. This way, the client can learn about the resource's support for resumable uploads and/or the upload URL. This is conceptually similar to how a client might wait for a 100 (Continue) interim response (see {{Section 10.1.1 of HTTP}}) before committing to work.
```

> A client might wait for a limited duration to receive a 104 (Upload Resumption Supported) interim response before starting to transmit the request content. This way, the client can learn about the resource's support for resumable uplaods and/or the upload URL. This is conceptually similar to how a client might wait for a 100 (Continue) interim response  (see {{Section 10.1.1 of HTTP}}) before committing to work. 

Author: ```suggestion
A client might wait for a limited duration to receive a 104 (Upload Resumption Supported) interim response before starting to transmit the request content. This way, the client can learn about the resource's support for resumable upload and/or the upload URL. This is conceptually similar to how a client might wait for a 100 (Continue) interim response  (see {{Section 10.1.1 of HTTP}}) before committing to work. 
```
